OpenRA.Mods.Common/UtilityCommands/UpdateMapCommand.cs

```diff
@@ -36,8 +36,7 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			var modData = Game.ModData = utility.ModData;
 
 			// HACK: We know that maps can only be oramap or folders, which are ReadWrite
-			var package = new Folder(Platform.EngineDir).OpenPackage(args[1], modData.ModFiles) as IReadWritePackage;
-			if (package == null)
+			if (!(new Folder(Platform.EngineDir).OpenPackage(args[1], modData.ModFiles) is IReadWritePackage package))
```

Comment:
This is a bit too ugly for my taste.
Can we at least keep a `var folder = new Folder()` before the if?